### ISSUE TITLE
Doesn't work for nested elements.

### ISSUE THREAD

**Issue by rahuljuneja** [viralpatel/jquery.shorten#22]
If you have nested elements within the main div. This code doesn't work.

**Comment by icraftzone** [comment]
Yes same problem here. It remove other tags like <p><br> etc. in between the contents. Any fix for it please!

**Comment by rhormazar** [comment]
I had a problem with nested elements too. It would hide everything including sibling h1 and p tags (changed their CSS to display: none) of the container that is selected for shortening.
